src/otk/transform.py

```diff
@@ -65,7 +65,6 @@ def resolve_dict(ctx: Context, state: State, tree: dict[str, Any]) -> Any:
         if isinstance(val, str):
             val = substitute_vars(ctx, val)
         if is_directive(key):
-
```

Comment:
This was added by precommit :( (which I still do not like very much!)

Reply:
Will be fixed after #125.